grafana/risingwave-dashboard.dashboard.py

```diff
@@ -1301,6 +1301,16 @@ def section_streaming_actors(outer_panels):
                         ),
                     ],
                 ),
+                panels.timeseries_percentage(
```

Comment:
Is it a percentage?

Reply:
Updated it to `Join Actor Match Duration Per Second`. Is it OK?